src/main/resources/site/parts/pifCalculator/pifCalculator.es6

```diff
@@ -14,6 +14,10 @@ const React4xp = __non_webpack_require__('/lib/enonic/react4xp')
 const {
   getLanguage
 } = __non_webpack_require__( '/lib/ssb/utils/language')
+const {
+  getCalculatorConfig, getKpiDatasetMonth
```

Comment:
Her skal du bruke getPifDataset for å finne siste tidspunkt i datasettet til pif

Reply:
endret